### PR TITLE
Improved audio session configuration for the noise sensor

### PR DESCRIPTION
**Pull request by jorishjanssen**
@Tubyhes could you QA and merge?

### CODE REVIEW

File: sense platform/sensors/CSNoiseSensor.m [hunk 1]
```diff
@@ -105,149 +111,24 @@ - (void) configureAudioSession
     AVAudioSession *session = [AVAudioSession sharedInstance];
     NSError *activationError = nil;
     NSError *categoryError = nil;
-
+	NSError *modeError = nil;
+	
     // audio session category
     NSString *appAudioSessionCategory = AVAudioSessionCategoryPlayAndRecord;
```
Comment (Tubyhes) [inline sense platform/sensors/CSNoiseSensor.m]: why not AVAudioSessionCategoryRecord?

File: sense platform/sensors/CSNoiseSensor.m [hunk 2]
```diff
@@ -105,149 +111,24 @@ - (void) configureAudioSession
     AVAudioSession *session = [AVAudioSession sharedInstance];
     NSError *activationError = nil;
     NSError *categoryError = nil;
-
+	NSError *modeError = nil;
```
Comment (Tubyhes) [inline sense platform/sensors/CSNoiseSensor.m]: why do we need 3 separate errors if we're not using them anyway?

File: sense platform/sensors/CSNoiseSensor.m (hunk 1 above)
Comment (jorishjanssen) [inline sense platform/sensors/CSNoiseSensor.m]: @Tubyhes documentation says it silences playback audio. That's not what we want...

File: sense platform/sensors/CSNoiseSensor.m (hunk 2 above)
Comment (jorishjanssen) [inline sense platform/sensors/CSNoiseSensor.m]: @Tubyhes they are currently being used in the error reporting NSLog statement.

**Pull request merged**